src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -79,6 +79,7 @@ public static void KeepAliveInterval_Roundtrips()
             AssertExtensions.Throws<ArgumentOutOfRangeException>("value", () => cws.Options.KeepAliveInterval = TimeSpan.MinValue);
         }
 
+        [ActiveIssue(27846)]
```

Comment:
Is this only failing on Win7?

Reply:
Yes, I only see the failure on Win7. I will narrow down the disable scope.